src/mesh/gmv_io.C

```diff
@@ -1046,7 +1042,11 @@ void GMVIO::write_ascii_old_impl (const std::string & fname,
           for (const auto & elem : mesh.active_element_ptr_range())
             {
               // Find the unique index for elem->subdomain_id(), print that to file
-              sbdid_map_iter map_iter = sbdid_map.find( elem->subdomain_id() );
+              auto map_iter = sbdid_map.find(elem->subdomain_id());
+
+              if (map_iter == sbdid_map.end())
+                libmesh_error_msg("Entry for subdomain " << elem->subdomain_id() << " not found.");
```

Comment:
Shouldn't this be an assert?  It should be currently guaranteed never to fail and if it ever does that'll be to a change causing an internal error.

Reply:
An assert would be fine with me... Before we were using the iterator without any checks so I figured something would be better than nothing.